tool/teleport/common/teleport.go

```diff
@@ -434,6 +435,15 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 
 	joinOpenSSH.Flag("debug", "Enable verbose logging to stderr.").Short('d').BoolVar(&ccf.Debug)
 
+	integrationCmd := app.Command("integration", "Integration commands")
```

Comment:
The integration is quite generic but what is the difference between :
-- teleprot discovery bootstrap
-- db configure bootstrap 

Do we need to introduce new patter for the `deployservice-iam`  service. 

Can we just create `teleport deployservice-iam bootstrap/configure` commands ?

Reply:
This is not (yet) discovery related.
It only installs a DatabaseService in Amazon ECS for the proxy.

We can, yeah
We might have other configuration needs when setting up the IAM requirements for other integrations and I was trying to create an abstraction/layer for them. So that they don't pollute the top level `teleport` commands